package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "lwc-monorepo",
+  "version": "2.23.1",
```

Comment:
Based on a conversation @divmain had with the #nucleus-talk [channel](https://salesforce-internal.slack.com/archives/C022MRLQ9R6/p1658273768701179) `version` is needed on the root `package.json` file.

I believe this will provide info to let nucleus identify which release is being made on the release comments, for [example](https://github.com/salesforce/lwc/pull/2980#issuecomment-1204693355).